dev-docs/modules/floors.md

```diff
@@ -1151,5 +1154,9 @@ If the currency function is unable to derive the correct cpm in any of the scena
 
 {: .table  }
 | Partners| Contact |
+<<<<<<< HEAD
```

Comment:
Please fix the merge conflict here

Reply:
I did not see a merge conflict. I was able to merge successfully